src/glibc.zig

```diff
@@ -1119,3 +1138,49 @@ pub fn needsCrtiCrtn(target: std.Target) bool {
         else => true,
     };
 }
+
+// Glibc v2.17 is the oldest supported linking target due to dependencies
+// from the Zig standard library.
+pub fn isSupportedGlibcVersion(ver: Version) bool {
+    return (ver.major > 2) or ((ver.major == 2) and (ver.minor >= 17));
+}
+
+// Check if given glibc version is supported by target's architecture.  Note that
+// target.zig's `available_libcs` should filter out many architectures before this.
+pub fn isArchSupportedGlibcVersion(target: std.Target, ver: Version) !void {
```

Comment:
this should just return a boolean and the error reporting can happen at the callsite. also it should not be `pub` unless it is called from outside this file.

Reply:
This function is gone now.